docs/conf.py

```diff
@@ -196,7 +196,11 @@
 .. raw:: html
 
     <div>
-      <p><a href="{{ env.docname.rsplit('/', 1).pop() }}.ipynb" class="btn">Download this notebook</a></p>
+      <p><a href="{{ env.docname.rsplit('/', 1).pop() }}.ipynb" class="btn">Download this notebook</a>
+      <br>Run the master version of this notebook:
```

Comment:
I wonder if this would be better all on one line, without the `<br>`?
```suggestion
      Run the master version of this notebook:
```